cogs/userinfo.py

```diff
@@ -58,33 +54,17 @@ def __init__(self, bot):
 
             await ctx.message.delete()
 
-    @userinfo.command()
-    async def avi(self, ctx, txt: str = None):
+    @commands.command(aliases=['avi'])
+    async def avatar(self, ctx, user: str=None):
```

Comment:
This command rename creates confusion between `avatar` and `avatars` (which is now an alias for `setavatar`).